src/Maui/Features/Creatures/Shared/CreatureDetail/CreatureProperty.razor

```diff
@@ -1,21 +1,23 @@
 ﻿@using Markdig;
 
 <MudItem md="4">
-    <MudTextField ReadOnly="ReadOnly" @bind-Value="@Title" />
+    <MudTextField T="string" ValueChanged="TitleChanged" ReadOnly="ReadOnly" Value="@Title" />
```

Comment:
I think the compiler can tell what your type is here - no longer need to specify T?

Reply:
You're right, this isn't needed anymore.
Removed!